### PR TITLE
PN-10310 SP Changes

### PR DESCRIPTION
**Pull request by abhishekjir**

### CODE REVIEW

File: PAS_Database/dbo/Stored Procedures/usprpt_GetWorkOrderAwaitingPartsReport.sql [hunk 1]
```diff
@@ -106,33 +114,35 @@ BEGIN
...
-		IF OBJECT_ID(N'tempdb..#tmpMultipleSubWOMStocklineKit') IS NOT NULL
-			DROP TABLE #tmpMultipleSubWOMStocklineKit
-
 		CREATE TABLE #AwaitingPartsData
 		(	
 			TotalRecordsCount BIGINT NULL, 
 			WorkOrderId BIGINT NULL, 
 			wonum VARCHAR(50) NULL, 
 			customername VARCHAR(500) NULL, 
 			woqnum VARCHAR(50) NULL,
```
Comment (rajeshgami93) [inline PAS_Database/dbo/Stored Procedures/usprpt_GetWorkOrderAwaitingPartsReport.sql]: Change the size as same as the WOQ table

File: PAS_Database/dbo/Stored Procedures/usprpt_GetWorkOrderAwaitingPartsReport.sql [hunk 2]
```diff
@@ -106,33 +114,35 @@ BEGIN
...
 			TotalRecordsCount BIGINT NULL, 
 			WorkOrderId BIGINT NULL, 
 			wonum VARCHAR(50) NULL, 
 			customername VARCHAR(500) NULL, 
 			woqnum VARCHAR(50) NULL, 
+			mpn VARCHAR(500) NULL, 
+			mpnDescription VARCHAR(500) NULL,
+			itemMasterId BIGINT NULL,
 			pn VARCHAR(500) NULL, 
 			pnDescription VARCHAR(500) NULL,
```
Comment (rajeshgami93) [inline PAS_Database/dbo/Stored Procedures/usprpt_GetWorkOrderAwaitingPartsReport.sql]: This is also needs to be change as same as master table

**Review by rajeshgami93 — CHANGES_REQUESTED** [review]
Please change

File: PAS_Database/dbo/Stored Procedures/usprpt_GetWorkOrderAwaitingPartsReport.sql (hunk 1 above)
Comment (abhishekjir) [reply PAS_Database/dbo/Stored Procedures/usprpt_GetWorkOrderAwaitingPartsReport.sql]: Resolved

File: PAS_Database/dbo/Stored Procedures/usprpt_GetWorkOrderAwaitingPartsReport.sql (hunk 2 above)
Comment (abhishekjir) [reply PAS_Database/dbo/Stored Procedures/usprpt_GetWorkOrderAwaitingPartsReport.sql]: Resolved

**Review by rajeshgami93 — APPROVED** [review]

**Pull request merged**